codegenerator/cli/templates/static/codegen/src/IO.res

```diff
@@ -247,15 +250,14 @@ module RollBack = {
             blockTimestamp: historyRow.current.block_timestamp,
           }
           switch historyRow.entityData {
-          | Set(entity) =>
+          | Set(entity: Entities.internalEntity) =>
+            setEntities->Utils.Dict.push((Entity.name :> string), entity.id)
```

Comment:
I'm not sure if just the id is enough info here. But lets see how it goes for now.